scvi/train/_trainingplans.py

```diff
@@ -1018,24 +984,26 @@ def training_step(self, batch, batch_idx):
     @partial(jax.jit, static_argnums=(0,))
     def jit_validation_step(
         self,
-        state: TrainState,
+        state: BatchTrainState,
         batch: Dict[str, jnp.ndarray],
         rngs: Dict[str, jnp.ndarray],
         **kwargs,
     ):
-        module = self.validation_module
-        rngs = {k: random.split(v)[1] for k, v in rngs.items()}
         vars_in = {"params": state.params, "batch_stats": state.batch_stats}
-        outputs = module.apply(vars_in, batch, rngs=rngs, **kwargs)
+        outputs = self.module.apply(vars_in, batch, rngs=rngs, **kwargs)
         loss_recorder = outputs[2]
         loss = loss_recorder.loss
         elbo = jnp.mean(loss_recorder.reconstruction_loss + loss_recorder.kl_local)
 
         return loss, elbo
 
     def validation_step(self, batch, batch_idx):
+        self.module.eval()
```

Comment:
we could probably call this once in the init right? I know it's not intensive, but probably is worth it.

Same for the train version

Reply:
ah I see it's cached behind the scenes, that's fine then.